### DISCUSSION TITLE
有没有用键值来取键名？

### DISCUSSION THREAD

**Discussion by lianbook** [Tencent/MMKV#1332] [Q&A]
用键值反取键名可以吗？？

**Comment by lingol** [comment]
It's impossible.

**Reply by lianbook** [reply]
我觉得可以增加这个功能，以键值取键名 的功能

**Reply by lingol** [reply]
Hi lianbook,

Thank you for taking the time to share your feature request and for your interest in MMKV. I appreciate your input and the thought you've put into suggesting this enhancement.

After careful consideration, I've decided not to pursue this feature at this time. While I understand its potential value, it doesn't align with the current direction and priorities of the project.

However, MMKV is open-sourced, and contributions from the community are always welcome. If this feature is particularly important to you, I encourage you to implement it and submit a pull request. I'd be happy to review it and consider it for inclusion.

Thank you again for your understanding and support.
